.ci_support/linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11python3.10.____cpython.yaml

```diff
@@ -28,6 +28,8 @@ pin_run_as_build:
     max_pin: x.x
 python:
 - 3.10.* *_cpython
+pytorch:
+- '2.4'
```

Comment:
I couldn't help but wonder if this is a bug in conda-smithy. PyTorch is not listed in the build/host/run sections, only test. So technically we should not need to list it in the pipeline configs. But it wouldn't hurt either..

Reply:
This likely originates in conda-build itself. That is how conda-smithy determines what keys to keep